BFF/src/main/kotlin/br/com/example/simple/screen/OnboardingScreen.kt

```diff
@@ -1,20 +1,78 @@
-package br.com.example.simple.screen
+package br.com.example.simple.builder
```

Comment:
Esse pacote deveria ser
```suggestion
package br.com.example.simple.screen
```